### ISSUE TITLE
Performance is worse than yours

### ISSUE THREAD

**Issue by vincentman** [huanghoujing/person-reid-triplet-loss-baseline#29]
Hi @huanghoujing ,
I try to train model on Market1501, but evaluation result is worse than yours.
Your score is mAP=75.29 and rank-1=89.04 on Market1501-S1, but my score is mAP=64.90 and rank-1=81.59. 

My training parameters are as following (refer to your description on github):
python3 script/experiment/train.py \
-d '(0,)' \
--only_test false \
--dataset market1501 \
--last_conv_stride 1 \
--normalize_feature false \
--exp_dir out \
--steps_per_log 10 \
--epochs_per_val 5

other parameters are default,
--trainset_part trainval
--lr_decay_type exp
--exp_decay_at_epoch 151
--total_epochs 300

I train model on Python 2.7 and Pytorch 0.3, and pip install with requirements. But I don't know why evaluation result is worse than yours. Could you give me some hints? Thanks.
